packages/babel-generator/test/fixtures/parentheses/break-statement/expected.js

```diff
@@ -0,0 +1,5 @@
+label: for (const a of [1, 2, 3]) {
+  break
+  /*test*/
```

Comment:
I know ideally this shouldn't be in a newline, but thanks to https://github.com/babel/babel/blob/d82afb407eaa4ff03b98b85824efcd161913952d/packages/babel-generator/src/printer.js#L532, newlines are always inserted before a BlockComment.

Why though - is it necessary to print newline always?